sense_api.py

```diff
@@ -67,67 +64,92 @@ def get_usage(s, sense_monitor_id, start ,granularity): # NOT WORKING
             }
         print payload
         #response = s.get('https://api.sense.com/apiservice/api/v1/app/history/usage?monitor_id=%s&granularity=$%s&start=%s&frames=%s' % (sense_monitor_id, 'MINUTE', valid_start, valid_frames), headers=headers)
-        response = s.get('https://api.sense.com/apiservice/api/v1/app/history/usage', headers=headers, data=payload)
+        response = s.get(API_URL+'app/history/usage', headers=headers, data=payload)
         return response
 
+    def get_realtime(self):
+        ws = create_connection("wss://clientrt.sense.com/monitors/%s/realtimefeed?access_token=%s" % (self.sense_monitor_id,  self.sense_access_token))
+        while True:
+            result = json.loads(ws.recv())
+            if 'payload' in result and not 'features' in result['payload']:
+                self._realtime = result['payload']
+                return self._realtime
+            
+    @property
+    def active_power(self):
+        if not self._realtime: self.get_realtime()
+        return self._realtime['w']
+    
+    @property
+    def active_solar_power(self):
+        if not self._realtime: self.get_realtime()
+        return self._realtime['solar_w']
```

Comment:
For those without Solar maybe something like this: `return self._realtime.get('solar_w', 'No Solar Found')`